### PR TITLE
Adds algoim recipe

### PR DESCRIPTION
**Pull request by ericneiva**
This is a dummy package that installs algoim's header files such that they are available to other packages.

### CODE REVIEW

File: A/algoim/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+cp -vr algoim/algoim/* "${includedir}/."
+"""
+
+platforms = [AnyPlatform()]
+
+products = Product[]
+
+dependencies = Dependency[]
+
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```
Comment (giordano) [inline A/algoim/build_tarballs.jl:22]:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)

```

File: A/algoim/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+version = v"0.1.0"
+sources = [
+    GitSource("https://github.com/algoim/algoim.git", "979cb3b7860b53751291352a929754d525b7fde1"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/
+install_license ${WORKSPACE}/srcdir/algoim/LICENSE
+mkdir -p "${includedir}"
+cp -vr algoim/algoim/* "${includedir}/."
```
Comment (giordano) [inline A/algoim/build_tarballs.jl:13]: Judging by the [documentation](https://algoim.github.io/), I think you should put the directory `algoim` itself in `${includedir}`, not its individual files directly:
```suggestion
cd ${WORKSPACE}/srcdir/algoim
install_license LICENSE
mkdir -p "${includedir}"
cp -vr algoim "${includedir}/."
```

**Review by giordano — COMMENTED** [review]
Thanks for your contribution!

**Comment by ericneiva** [comment]
Thanks for your suggestions! All agreed.

**Pull request merged**